tests/data/hostgaplugin/ext_conf-requested_version.xml

```diff
@@ -60,7 +60,7 @@
   "runtimeSettings": [
     {
       "handlerSettings": {
-        "protectedSettingsCertThumbprint": "4037FBF5F1F3014F99B5D6C7799E9B20E6871CB3",
+        "protectedSettingsCertThumbprint": "BD447EF71C3ADDF7C837E84D630F3FAC22CCD22F",
```

Comment:
The older test data is encrypted using an algorithm that is no longer supported on OpenSSL 3.0. All the changes in the data directory are to update the test data with new certificates pulled from a current live VM.

You will see updates of the certificate thumbprints throughout this PR, as well as the certificate data itself.